src/ui/public/notify/notifier.js

```diff
@@ -314,6 +315,27 @@ Notifier.prototype.info = function (msg, cb) {
   }, cb);
 };
 
+/**
+ * Display a banner message
+ * @param  {String} msg [description]
+ * @return {[type]}     [description]
+ */
+Notifier.prototype.banner = function (msg, cb) {
+  if (this._lastBanner) {
```

Comment:
Rather than implement this as a one-off for banner, how about implementing a private `set` function that handles this on a type by type basis.  So once that is implemented, your banner function becomes something like:

```js
Notifier.prototype.banner = function (msg, cb) {
  return set({
    type: 'banner',
    markdown: msg,
    lifetime: Notifier.config.bannerLifetime,
    actions: ['accept']
  }, cb);
}
```